app.rb

```diff
@@ -90,5 +91,5 @@ def verify_signature(payload_body)
 end
 
 def redmine_issue_repos
-  ENV['REDMINE_ISSUE_REQUIRED_REPOS'].to_s.split.map { |r| Regexp.new("\\A#{r}\\z") }
+  YAML.load_file('config/redmine_issue_required_repos.yaml')
```

Comment:
If we change this line:

```
   pull_request.check_commits_style if redmine_issue_repos.find { |r| pull_request.repo.match(r) } && pr_action != 'created'
```
to
```
   pull_request.check_commits_style if redmine_issue_repos.include?(pull_request.repo) && pr_action != 'created'
```

This should match only on explicit names since the `redmine_issue_repos` is an array. For example, 'theforeman/foreman_docker' won't resolve if 'theforeman/foreman' is provided.